homeassistant/components/smarttub/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/smarttub",
   "iot_class": "cloud_polling",
   "loggers": ["smarttub"],
-  "requirements": ["python-smarttub==0.0.38"]
+  "requirements": ["python-smarttub==0.0.39"]
```

Comment:
Please bump in a separate PR

Reply:
https://github.com/home-assistant/core/pull/139313